docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.3'
+version: '2.0'
```

Comment:
version 3.3 is on purpose since it support's caching of volumes which speeds up the development on mac/windows.

Reply:
What is the benefit of setting it to 2.0?

Reply:
When i installed on fresh system docker libraries, it wasn't supporting version 3
And i left it accidentaly, will get back that